tests/unit/sagemaker/huggingface/test_estimator.py

```diff
@@ -219,6 +218,7 @@ def test_huggingface(
     huggingface_training_version,
     huggingface_pytorch_training_version,
 ):
+
```

Comment:
can you get rid of this line here? 

We are not changing anything in this file right?

Reply:
updated.